lib/rom/setup/auto_registration.rb

```diff
@@ -10,7 +10,7 @@ class AutoRegistration
 
     include Options
 
-    option :namespace, reader: true, type: [TrueClass, FalseClass], default: true
+    option :namespace, reader: true, type: [TrueClass, FalseClass, String], default: true
```

Comment:
Meaning that we want to change the api to accept `allow:` instead of `type:`?
Like so:
`allow: true`
`allow: false`
`allow: 'My::Namespace'`

?